### DISCUSSION TITLE
Wireless Wiring Guide

### DISCUSSION THREAD

**Discussion by zachserviss** [juhakaup/keyboards#43] [Q&A]
Hey got all the parts for a wireless build and looking for some guidance on wiring with a battery and power switch. Am I missing a doc somewhere?

**Comment by juhakaup** [comment]
There is no specific wireless build guide.
Wire one of the battery leads directly to the controller and the other to the center prong of the switch.
Then wire one of the remaining prongs to from the switch to the controller.

**Reply by zachserviss** [reply]
Thank you for your reply! I'm a visual learner so just want to make sure this is correct:

![image](https://github.com/user-attachments/assets/312c158b-1e8c-4043-8f5e-34e4dd8a1622)

Thanks again!

**Reply by juhakaup** [reply]
Looks good to me.
